router.php

```diff
@@ -50,11 +50,11 @@
 (function () {
     $target = getenv('FUNCTION_TARGET', true);
     if (false === $target) {
-        throw new RuntimeException('FUNCTION_TARGET is not set');
+        $target = 'function';
     }
     $signatureType = getenv('FUNCTION_SIGNATURE_TYPE', true);
     if (false === $signatureType) {
-        throw new RuntimeException('FUNCTION_SIGNATURE_TYPE is not set');
+        $signatureType = 'http';
```

Comment:
Would be better to just use a ternary statement here:

```php
$signatureType = getenv('FUNCTION_SIGNATURE_TYPE', true) ?: 'http';
```

Reply:
Done, thanks.